website/pages/docs/guides/using-scarb-in-ci.mdx

```diff
@@ -17,40 +18,65 @@ env:
 
 jobs:
   build:
-    name: test ${{ matrix.name }}
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        include:
-          - name: linux x86-64
-            os: ubuntu-latest
-          - name: macos x86-64
-            os: macos-latest
-          - name: windows x86-64
-            os: windows-latest
+    name: build
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
 
       - name: Setup Scarb for Linux
-        if: matrix.os == 'ubuntu-latest'
         run: |
           wget https://github.com/software-mansion/scarb/releases/download/${{ env.SCARB_VERSION }}/scarb-${{ env.SCARB_VERSION }}-x86_64-unknown-linux-gnu.tar.gz
           tar -xvzf scarb-${{ env.SCARB_VERSION }}-x86_64-unknown-linux-gnu.tar.gz
           echo "$GITHUB_WORKSPACE/scarb-${{ env.SCARB_VERSION }}-x86_64-unknown-linux-gnu/bin" >> $GITHUB_PATH
 
-      - name: Setup Scarb for OSX
-        if: matrix.os == 'macos-latest'
-        run: |
-          wget https://github.com/software-mansion/scarb/releases/download/${{ env.SCARB_VERSION }}/scarb-${{ env.SCARB_VERSION }}-x86_64-apple-darwin.tar.gz
-          tar -xvzf scarb-${{ env.SCARB_VERSION }}-x86_64-apple-darwin.tar.gz
-          echo "$GITHUB_WORKSPACE/scarb-${{ env.SCARB_VERSION }}-x86_64-apple-darwin/bin" >> $GITHUB_PATH
+      - run: scarb build
+```
 
-      - name: Setup Scarb for Windows
-        if: matrix.os == 'windows-latest'
-        run: |
-          C:\msys64\usr\bin\wget.exe https://github.com/software-mansion/scarb/releases/download/${{ env.SCARB_VERSION }}/scarb-${{ env.SCARB_VERSION }}-x86_64-pc-windows-msvc.zip
-          7z x scarb-${{ env.SCARB_VERSION }}-x86_64-pc-windows-msvc.zip
-          Add-Content $env:GITHUB_PATH "$env:GITHUB_WORKSPACE/scarb-${{ env.SCARB_VERSION }}-x86_64-pc-windows-msvc/bin"
+## GitlabCI
+
+You can find an example of the Scarb setup in the following gitlab ci configuration.
```

Comment:
```suggestion
You can find an example of the Scarb setup in the following GitLab CI configuration.
```

try to use with correct casing everywhere possible :) https://docs.gitlab.com/ee/ci/